examples/double-proxy/docker-compose.yaml

```diff
@@ -0,0 +1,66 @@
+version: "3.7"
+services:
+
+  proxy-frontend:
+    build:
+      context: .
+      dockerfile: Dockerfile-proxy
+    networks:
+      edge:
+    ports:
+      - "10000:10000"
+
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile-app
+    networks:
+      edge:
+      postgres-frontend:
+
+  proxy-postgres-frontend:
+    build:
+      context: .
+      dockerfile: Dockerfile-proxy-frontend
+    networks:
+      postgres-frontend:
+        aliases:
+          - postgres
+      postgres-in-between:
+        aliases:
+          - proxy-postgres-frontend.example.com
```

Comment:
i dont think this is strictly necessary - im not sure if the server checks the clients dns <> cert